libminifi/include/Exception.h

```diff
@@ -41,12 +41,13 @@ enum ExceptionType {
   SITE2SITE_EXCEPTION,
   GENERAL_EXCEPTION,
   REGEX_EXCEPTION,
+  OPC_EXCEPTION,
   MAX_EXCEPTION
 };
 
 // Exception String 
 static const char *ExceptionStr[MAX_EXCEPTION] = { "File Operation", "Flow File Operation", "Processor Operation", "Process Session Operation", "Process Schedule Operation", "Site2Site Protocol",
-    "General Operation", "Regex Operation" };
+    "General Operation", "Regex Operation", "OPC Operation" };
```

Comment:
Opc exception should be moved into ooc extension, and there should be no pollution in core

Reply:
I agree, there should be no pollution in the core. 

I moved the exception to the extension, but it's not so nice as Exception class in MiNiFi was not designed for being extended in any extension. (virtual functions? protected members?)

Because of this I find your comment a bit belated and misguided. Should had been made by the time such a class was designed. 

This topic definitely worth a follow-up ticket,  I'm open to discussions, suggestions.